WeatherApp-UIKit/WeatherApp-UIKit/WeatherViewModelState.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  WeatherViewModelState.swift
+//  WeatherApp-UIKit
+//
+//  Created by Jozef Matus on 11/03/2020.
+//  Copyright © 2020 Industrial Binaries. All rights reserved.
+//
+
+import Foundation
+
+enum WeatherViewModelState: Equatable {
+  case loading
+  case loaded(
+    weatherDescription: String,
+    temperature: String,
+    icon: String,
+    location: String,
+    weatherFeedback: Bool?
+  )
+  case error
+}
+
+func == (lhs: WeatherViewModelState, rhs: WeatherViewModelState) -> Bool {
```

Comment:
This overload is here to provide a custom implementation of `==` that doesn't compare the `weatherFeedback` value?? Why?

Reply:
It's a leftover, not used in the code. Nice catch!